File: strings/findlongestword.py
def findlongestword(str):
    stringsplit = str.split(" ")
    longestword = 0
    for s in stringsplit:
        if len(s) > longestword:
            longestword = len(s)

    return longestword
print(findlongestword("There's a double rainbow in the sky")) # 7
print(findlongestword("I like mushrooms and bagels")) # 9
print(findlongestword("Sweet serendipity")) # 11

File: strings/findlongestword.js
function findlongestword(str){
    let longestword = 0
    let stringsplit = str.split(" ")
    for(var i = 0; i < stringsplit.length; i++){
        if(stringsplit[i].length > longestword){
            longestword = stringsplit[i].length
        }
    }
    return longestword

}
console.log(findlongestword("My cat likes to eat bananas")) //7
console.log(findlongestword("The water is wet")) //5
console.log(findlongestword("My favorite color is red")) //8
